.github/workflows/pypi.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.10
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install build twine
+        python -m build
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@master
```

Comment:
Why do you use `@master` version? :thinking: 
in this [documentation](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries), the `@v1` is used. In the same documentation, they refer to an example [python-publish](https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml) where `@27b31702a0e7fc50959f5ad993c78deac1bdfc29` is used.

Reply:
you are right, on their [GH page](https://github.com/pypa/gh-action-pypi-publish) the `release/v1` version is used. Will do that one.